helmfile/charts/gatekeeper/constraints/templates/disallow-tag/constraint.yaml

```diff
@@ -21,6 +21,12 @@ spec:
         operator: NotIn
         values:
         - operator
+      - key: kubernetes.io/metadata.name
```

Comment:
Shouldn't be, but I think this will impact Gatekeeper audit as well.

Do we want to exclude them from the webhooks? Do we depend/expect that it is covered by PSPs?

Reply:
Talked with viktor and we decided that it was best to just keep the exemptions on the webhooks. This seem to be something that we actually want especially since we did remove the ignore label from the kube-system namespace [here](https://github.com/elastisys/compliantkubernetes-apps/commit/4b3368fd41a7e9226d1684c4865b372ad095026d), which means that the webhooks will interfere with kube components if gatekeeper is down.

Reply:
Yes, that sounds like the better option.